docs/releases/1.11.txt

```diff
@@ -562,6 +562,11 @@ Miscellaneous
 * ``ConditionalGetMiddleware`` no longer sets the ``Date`` header as Web
   servers set that header.
 
+* To simplify Django's timezone handling, ``pytz`` is a required dependency.
+  It's automatically installed along with Django. Also, support for
+  ``settings.TIME_ZONE=None`` is removed as the behavior isn't commonly used
+  and is questionably useful.
```

Comment:
What about giving upgrade instructions? Perhaps mention the workaround you described on the mailing list to discover the system time zone (with a dependency whose name I don't remember)?